rflx/pyrflx/typevalue.py

```diff
@@ -123,19 +137,18 @@ def _last(self) -> int:
         assert isinstance(last, Number)
         return last.value
 
-    def assign(self, value: int, check: bool = True) -> None:
+    def assign(self, value: int, length: int = 0, check: bool = True) -> None:
         if (
-            self._type.constraints("__VALUE__", check).simplified(
+            self._type.constraints("__VALUE__", True).simplified(
```

Comment:
The parameter `check` seems not to be used at all.

Reply:
Fixed by 9f39fdc